src/components/Image/Image-story.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import Component from './Image';
+
+const props = {
+  src: 'https://assets.imgix.net/unsplash/mountains.jpg'
+};
+
+storiesOf('Image', module).addWithJSX('Default', () => <Component {...props} />);
```

Comment:
```suggestion
storiesOf('Image', module).add('Default', () => <Component {...props} />);
```
use `add` after rebase for the new storybook version